hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/BucketEndpoint.java

```diff
@@ -507,6 +514,10 @@ public Response putAcl(String bucketName, HttpHeaders httpHeaders,
       LOG.error("Error in set ACL Request for bucket: {}", bucketName,
           exception);
       throw exception;
+    } catch (OS3Exception ex) {
+      getMetrics().incPutAclFailure();
+      LOG.debug("Failed to put acl of Bucket " + bucketName, ex);
```

Comment:
We don't need the debug message, because the exception is logged at debug level in `newError`.

https://github.com/apache/ozone/blob/e81ab0a3f76b09072c23e0151c5e28673623ed6c/hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/exception/S3ErrorTable.java#L141-L145

```suggestion
```

Reply:
Will make the change, thanks !!